src/hokusai/hokusai.yml

```diff
@@ -55,11 +55,16 @@ commands:
 jobs:
   test:
     executor: deploy
+    parameters:
+      filename:
+        type: string
+        default: ./hokusai/test.yml
+        description: The docker-compose yaml file to use
     steps:
       - setup-docker
       - run:
           name: Test
-          command: hokusai test
+          command: hokusai test -f << parameters.filename >>
```

Comment:
What happens if you pass nothing?

Reply:
It uses the default as defined by the parameter block above. That default is the same as Hokusai’s.